packages/protocol/test/bridge/libs/LibBridgeProcess.test.ts

```diff
@@ -0,0 +1,296 @@
+import { expect } from "chai"
+import hre, { ethers } from "hardhat"
+import { Message } from "../../utils/message"
+import { AddressManager, Bridge } from "../../../typechain"
+import { getSlot } from "../../../tasks/utils"
+const helpers = require("@nomicfoundation/hardhat-network-helpers")
+// ethers.provider.getBalance(address)
+
+describe("LibBridgeProcess", function () {
+    async function deployLibBridgeProcessFixture() {
+        const [owner, nonOwner, etherVaultOwner] = await ethers.getSigners()
+
+        // slot number of IBridge.State for TestLibBridgeProcess.
+        // mapping destChains is at position 0 (201)
+        // mapping messageStatus is at position 1 (202)
+        // nextMessageId is at position 2 (203)
+        // Context takes up 3 slots, starts at position 3 (204)
+        const stateSlot = 201
```

Comment:
Its more recommended to always get the `IBridge.State` slot number dynamically from the latest build info, then if the `LibBridgeProcess` contract changed, we wont need to accordingly change the hard-coded slot here, e.g:
```js
function getStateSolt() {
    const buildInfoDir = path.join(__dirname, "../../../artifacts/build-info")
    const contractPath = "contracts/test/bridge/libs/TestLibBridgeProcess.sol"
    const contractName = "TestLibBridgeProcess"

    for (const buildInfoJson of fs.readdirSync(buildInfoDir)) {
        const { output } = require(path.join(buildInfoDir, buildInfoJson))

        if (!output.contracts[contractPath]) continue

        const slotInfo = output.contracts[contractPath][
            contractName
        ].storageLayout.storage.find(({ label }: any) => label === "state")

        if (slotInfo) return Number(slotInfo.slot)
    }
}
```